GitCommands/RevisionReader.cs

```diff
@@ -159,13 +108,14 @@ public sealed class RevisionReader : IDisposable
             }
         }
 
-        private ArgumentBuilder BuildArguments(int maxCount,
+        public static ArgumentBuilder BuildArguments(int maxCount,
```

Comment:
Just the method for `ParentsAreRewritten` could be exposed though, but I found that it was easier to see what the RevisionReader was doing this way.This could be moved to GitCommandHelper.cs but the format is internal to the RevisionReader.